### ISSUE TITLE
[Feature Request] - namespace css variables please

### ISSUE THREAD

**Issue by nawlbergs** [goetzrobin/spartan#339]
### Which scope/s are relevant/related to the feature request?

Don't know / other

### Information

Maybe I'm missing something... but CSS variables (eg: --background, --card) should be name spaced to avoid conflict with other frameworks or local variables.

Eg: --sptn-background, --sptn-card.

### Describe any alternatives/workarounds you're currently using

hope no other frameworks/libs use similar variables.

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by goetzrobin** [comment]
I like this idea. We have also talked about adding a prefix to the spartan tailwind classes, so I can see how it makes sense for CSS, too. @benpsnyder, I am curious if you guys are already using a similar approach

**Comment by benpsnyder** [comment]
Yes my team would be ready to support prefixes as well for tailwind classes. @goetzrobin you decide on the prefix and I can schedule getting a PR done for TW
